spec/helpers.lua

```diff
@@ -391,9 +434,16 @@ end
 -- @function proxy_ssl_client
 -- @param timeout (optional, number) the timeout to use
 -- @param sni (optional, string) the sni to use
-local function proxy_ssl_client(timeout, sni)
+-- @param http_version (optional, number) the http version to use
+local function proxy_ssl_client(timeout, sni, http_version)
   local proxy_ip = get_proxy_ip(true, true)
   local proxy_port = get_proxy_port(true, true)
+  local ihttp_version
```

Comment:
What does the prefix `i` mean? Could we have a better name?

Reply:
`i` means `inner`, to avoid the same variable name with the last function parameters. Do you have a better name in here?

Reply:
If it is only used in `opts`, how about `opts_http_version`?

Reply:
done